### DISCUSSION TITLE
eb init 플랫폼 설정

### DISCUSSION THREAD

**Discussion by MinHongSun** [fsoftwareengineer/todo-application#123] [질문과 답변] [answered]
eb init 환경설정중  플랫폼 프런치에서 java 8 running on 64bit amazion linux 선택지가 없더군요.
찾아보니 AWS 내에서 JAVA SE 플랫폼 버전이 올해 6월 30일부로 만료되었다고합니다.

 출처 : https://docs.aws.amazon.com/ko_kr/elasticbeanstalk/latest/platforms/platforms-retiring.html

이렇게되면 프로덕션 배포 실습을 할 수 없는 것인지 궁굼합니다.

**Comment by fsoftwareengineer** [comment, marked as answer]
Correto 8으로 진행하시면 됩니다. Correto도 자바에요.

**Reply by MinHongSun** [reply]
감사합니당!
